src/comm.cpp

```diff
@@ -90,7 +90,9 @@ void comm::sendRawPacket(std::array<uint8_t, 64> &tx_data)
 void comm::sendString(std::string tx_string)
 {
     tx_string.append(64 - tx_string.length(), ' ');
-    Serial.write(tx_string.c_str(), 64);
+    Serial.write(tx_string.c_str(), 64); // Read by Serial Mon as ASCII
+    Serial.send_now();
+    Serial.write("\n");
```

Comment:
Add newline to earlier packet, rather than as a separate packet? Reduces latency